inlong-dashboard/src/metas/sinks/defaults/Hive.ts

```diff
@@ -117,39 +117,33 @@ export default class HiveSink extends SinkInfo implements DataWithBackend, Rende
   password: string;
 
   @FieldDecorator({
-    type: 'input',
-    rules: [{ required: true }],
-    props: values => ({
-      disabled: [110, 130].includes(values?.status),
-      placeholder: 'jdbc:hive2://127.0.0.1:10000',
-    }),
-  })
-  @I18n('JDBC URL')
-  jdbcUrl: string;
-
-  @FieldDecorator({
-    type: 'input',
-    rules: [{ required: true }],
-    tooltip: i18n.t('meta.Sinks.DataPathHelp'),
```

Comment:
This has been removed, but I see that node's dictionary key specification is used incorrectly, can you help to modify it?

See: (`meta.Sinks.` prefix is wrong)
https://github.com/apache/inlong/blob/master/inlong-dashboard/src/metas/nodes/defaults/Hive.ts#L41